### DISCUSSION TITLE
Can jte support XML templates out of the box?

### DISCUSSION THREAD

**Discussion by thadguidry** [casid/jte#389] [General]
or is there an extension needed?

**Comment by edward3h** [comment]
Can you be more specific about what you are trying to do?

You can use plain text content type to produce XML (or any other text type), but you won't get the validation that works with HTML.

If you are looking to serialize data as XML you would probably be better off with one of the Java libraries designed to do that though. [Here's a Baeldung article listing some](https://www.baeldung.com/java-xml).

**Reply by thadguidry** [reply]
Great!  I'll use the plain text content type to generate the XML.  Thanks much!
